src/main/java/core/basesyntax/ReadData.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadData {
+    public String getDataFromFile(String fromFile) {
```

Comment:
bad class name - you should not use verbs

how about `FileDataReader`

and considering that you could simplify the method name to `public String readFrom(String fromFile)`